src/main/java/org/apache/pulsar/io/jcloud/credential/GcsCredential.java

```diff
@@ -18,10 +18,10 @@
  */
 package org.apache.pulsar.io.jcloud.credential;
 
-import java.io.IOException;
+import static org.apache.pulsar.io.jcloud.BlobStoreAbstractConfig.PROVIDER_GCS;
```

Comment:
normally static import will be at the last of all the imports

and the whole imports should be formatted

Reply:
fixed.